internal/projection/process/event_enricher_test.go

```diff
@@ -299,6 +299,37 @@ var _ = Describe("Process message", func() {
 			assertHostsSummary(enrichedEvent)
 		})
 	})
+	When("Enrich a cluster event with different release tag locations", func() {
+		It("with release tag as metadata", func() {
+			myMetadata := map[string]interface{}{
+				"versions": map[string]interface{}{
+					"release_tag": "foobar",
+					"versions":    `{"assisted-installer": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-rhel8:latest", "assisted-installer-controller": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-reporter-rhel8:latest", "assisted-installer-service": "quay.io/app-sre/assisted-service:880ad10", "discovery-agent": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-agent-rhel8:latest"}`,
+				},
+			}
+			message := "Cluster with ID myid updated"
+			event := getEventWithMetadata("cluster_updated", message, myMetadata)
+			cluster := map[string]interface{}{
+				"foo": "bar",
+			}
+			hosts := getHosts()
+			infraEnvs := getInfraEnvs()
+			enrichedEvent := enricher.GetEnrichedEvent(event, cluster, hosts, infraEnvs)
+			Expect(*enrichedEvent.ReleaseTag).To(Equal("foobar"))
+		})
+
+		It("with no release tag", func() {
```

Comment:
Let's add a test with the older versions format to check if the code would work also in that case

Reply:
i.e.
```
myMetadata := map[string]interface{}{
					"versions":    `{"assisted-installer": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-rhel8:latest", "assisted-installer-controller": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-reporter-rhel8:latest", "assisted-installer-service": "quay.io/app-sre/assisted-service:880ad10", "discovery-agent": "registry-proxy.engineering.redhat.com/rh-osbs/openshift4-assisted-installer-agent-rhel8:latest"}`,
			}
```